### PR TITLE
Datadog support

### PR DESCRIPTION
**Pull request by ja8zyjits**
### Issue Reference
This PR addresses the Issue : Fixes #

### Summarize
It will enable datadog support in production
but it is in docker, someone patch this for heroku

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -29,4 +30,22 @@ services:
...
-      
+      # - datadog
+  # agent section
+  datadog:
+    image: datadog/docker-dd-agent
+    restart: always
+    ports:
+      - "8126:8126"
+      - "8125:8125/udp"
+    links:
```
Comment (r3dsm0k3) [inline docker-compose.yaml]: links are legacy feature and it is deprecated. We already have a network between containers.

Comment (ja8zyjits) [reply docker-compose.yaml]: no issues, the deployment is in heroku, so this wont affect

Comment (r3dsm0k3) [reply docker-compose.yaml]: Why not remove it then?

File: floodrelief/settings.py [hunk 2]
```diff
@@ -71,8 +72,24 @@ def get_list(text):
...
     'rest_auth',
+    'ddtrace.contrib.django',
 ]
 
+DATADOG_TRACE = {
+    'DEFAULT_SERVICE': 'django',
+    'TAGS': {'env': 'production'},
+    'AGENT_HOSTNAME': env('DATADOG_HOST'),
+    'AGENT_PORT': '8126',
+    'ENABLED': True,
```
Comment (r3dsm0k3) [inline floodrelief/settings.py]: Can we disable the tracing for dev based on an env?

Comment (ja8zyjits) [reply floodrelief/settings.py]: sure, i believe this is shelved for now. so it wont reflect in development now

File: docker-compose.yaml (hunk 1 above)
Comment (ja8zyjits) [reply docker-compose.yaml]: I haven't tested without links, shall do it later. A bit backwards in awareness with docker features.

**Comment by biswaz** [comment]
Implemented on #912

**Pull request closed without merging**